### PR TITLE
[IMP] Add handle widget in acceptability criteria view. Added unittest sequence for allocation criteria and created migration file for the sequence field

### PR DESCRIPTION
**Pull request by egrivero**

### CODE REVIEW

File: user_story/model/user_story.py [hunk 1]
```diff
@@ -644,13 +644,15 @@ def _get_default_sequence(self, cr, uid, context=None):
...
+            if order[-1][2]['sequence_ac'] > len(ac_ids):
+                seq = order[-1][2]['sequence_ac'] + 1
         elif len(ac_ids):
             order = [x[1] for x in ac_ids]
             read_ac = self.read(cr, uid, order,
                                 ['sequence_ac'], context=context)
             maxi = max(read_ac, key=lambda x: x['sequence_ac'])
-            seq = maxi['sequence_ac'] + 1
+            if maxi['sequence_ac'] > len(ac_ids):
+                seq = maxi['sequence_ac'] + 1
```
Comment (nhomar) [inline user_story/model/user_story.py]: seq = maxi['sequence_ac'] + 1 --> no
maxi['sequence_ac'] += 1 --> si

File: user_story/model/user_story.py [hunk 2]
```diff
@@ -644,13 +644,15 @@ def _get_default_sequence(self, cr, uid, context=None):
         seq = len(ac_ids) + 1
         if in_memory:
             order = sorted(in_memory, key=lambda x: x[2]['sequence_ac'])
-            seq = order[-1][2]['sequence_ac'] + 1
+            if order[-1][2]['sequence_ac'] > len(ac_ids):
+                seq = order[-1][2]['sequence_ac'] + 1
```
Comment (nhomar) [inline user_story/model/user_story.py]: Idem.
seq = order[-1][2]['sequence_ac'] + 1 ->
order[-1][2]['sequence_ac'] += 1

**Comment by moylop260** [comment]
Please add a name of pull request more explicit

**Comment by moylop260** [comment]
You can ignore the following lint `user_story/model/user_story.py:84: [E1136(unsubscriptable-object), UserStory.write] Value 'criteria[1]' is unsubscriptable`!

Could you fix the following lint `user_story/tests/test_user_story.py:71:32: E126 continuation line over-indented for hanging indent`?

**Comment by egrivero** [comment]
Ready, corrected lint

**Comment by moylop260** [comment]
Could you create the dummy pr to vx/instance?

**Comment by egrivero** [comment]
yes, Vauxoo/instance#280

**Comment by moylop260** [comment]
2 days ago was build
Could you rebuild again?

**Comment by moylop260** [comment]
👍 
The red of travis is unrelated for this focus.

**Comment by gquilarque** [comment]
@egrivero please remember add a description for this PR.

+1 Functional Test.

**Comment by zaoral** [comment]
Excellent!
Please dont forget to also update `user_story/i18n/user_story.pot`

**Comment by egrivero** [comment]
Ready update in user_story.pot

**Pull request merged**